packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -106,20 +106,20 @@ export function applyFieldOverrides(options: ApplyFieldOverrideOptions): DataFra
     const fields: Field[] = frame.fields.map(field => {
       // Config is mutable within this scope
       const fieldScopedVars = { ...scopedVars };
-      const title = getFieldTitle(field, frame, options.data);
+      const displayName = getFieldDisplayName(field, frame, options.data);
 
       fieldScopedVars['__field'] = {
         text: 'Field',
         value: {
-          name: title, // Generally appropriate (may include the series name if useful)
+          name: displayName, // Generally appropriate (may include the series name if useful)
           labels: formatLabels(field.labels!),
           label: field.labels,
         },
       };
 
       field.state = {
         ...field.state,
-        title: title,
+        displayName: displayName,
```

Comment:
nit: maybe change to
```suggestion
        displayName,
```